sdk/monitor/azure-monitor-opentelemetry/tests/configuration/test_util.py

```diff
@@ -36,25 +36,35 @@
     "test.attributes.1": "test_value_1",
     "test.attributes.2": "test_value_2"
```

Comment:
I think we also need to add a test in `test_configure.py` to test the new param

Reply:
Since the kwargs only go to utils, there's not much to do, but I added the value in a few more places

Reply:
I think the test would be to verify that each signal's provider did indeed get instantiated with the resource instance that was passed in.

Reply:
Already done before this PR.